### ISSUE TITLE
Multple UV layers

### ISSUE THREAD

**Issue by RavvickHipnot** [Facepunch/RustWorkshop#159]
Some skins, e.g. the Double Barrel shotgun, have multiple UV layers, which arrive in Substance Painter, Photoshop etc. as one map. This makes painting individual UVs very difficult. Is there any chance of providing extra meshes in a format that's better to use?

**Comment by k-lj** [comment]
Agreed, there are overlapping UV's have been a big problem for me too.
